src/modules/ROOT/pages/rfc/federation/authentication.adoc

```diff
@@ -28,13 +66,37 @@ Node A creates a new node and generates a pair URI::
 `token` is a generated OTT token that gives us a basic way of authenticating the initial requests that are performed outside of the authentication facility.
 ====
 
+[source,bash]
+----
+curl -X POST "$API_BASE/federation/nodes/" \
+  -H "authorization: Bearer $JWT_FD" \
+  --header "Content-Type: application/json" \
+  --data "{
+    \"sharedDomain\": \"$DOMAIN_FD_ORIGIN\",
```

Comment:
Why is this sent in the payload? If this is our/current domain, backend should be able to get to it

Reply:
As agreed, this will be later provided via configuration. For now, this will be used while developing.